### PR TITLE
Run selected vRouter agent's unit tests

### PR DESCRIPTION
**Pull request by aethyrxeo**

### CODE REVIEW

File: CIScripts/Build/BuildFunctions.ps1 [hunk 1]
```diff
@@ -392,10 +381,15 @@ function Invoke-AgentTestsBuild {
...
         [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
-            "", Justification="TASK_UTIL_RETRY_COUNT is used agent tests for determining " +
+            "", Justification="TASK_UTIL_RETRY_COUNT is used in agent tests for determining " +
             "timeout's threshold. They were copied from Linux unit test job.")]
         $Env:TASK_UTIL_RETRY_COUNT = 6000
 
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "", Justification="CONCURRENCY_CHECK_ENABLE is used in agent tests." +
+            " They were copied from Linux unit test job.")]
+        $Env:CONCURRENCY_CHECK_ENABLE = "true"
```
Comment (jacek-izykowski) [inline CIScripts/Build/BuildFunctions.ps1]: Once `https://review.opencontrail.org/#/c/49774` is merged, this is no longer necessary to support contrail-common tests. Which tests actually need is?

File: CIScripts/Build/BuildFunctions.ps1 [hunk 2]
```diff
@@ -406,12 +400,17 @@ function Invoke-AgentTestsBuild {
             "vnsw\agent\test",
             "xml\test",
             "xmpp\test"
-        ) | ForEach-Object { "$rootBuildDir\$_" }
+        ) | ForEach-Object { Join-Path $rootBuildDir $_ }
+
+        $AgentExecutables = Get-ChildItem -Recurse $TestsFolders | Where-Object {$_.Name -match '.*?\.exe$'}
+        # TODO: Some unit tests of Agent need DLLs present in the same
```
Comment (jacek-izykowski) [inline CIScripts/Build/BuildFunctions.ps1]: Do we build those "some unit tests"?

Comment (aethyrxeo) [reply CIScripts/Build/BuildFunctions.ps1]: Not now, but in future yes.

File: CIScripts/Build/BuildFunctions.ps1 (hunk 1 above)
Comment (aethyrxeo) [reply CIScripts/Build/BuildFunctions.ps1]: Actually only that one it seems. Gonna delete it.

**Comment by aethyrxeo** [comment]
recheck no bug

**Pull request merged**